hawkular-alerts-rest-tests/pom.xml

```diff
@@ -219,6 +226,7 @@
               <excludes>
                 <excludeFile>org/hawkular/alerts/rest/ClusterITest.class</excludeFile>
                 <excludeFile>org/hawkular/alerts/rest/PerfCrudITest.class</excludeFile>
+                <excludeFile>org/hawkular/alerts/rest/CORSITest.class</excludeFile>
```

Comment:
It's a necessary exclusion for -Pitest-external because the tests rely on a specially configured test server.